go/vt/sqlparser/analyzer.go

```diff
@@ -60,6 +60,8 @@ func Preview(sql string) int {
 		firstWord = trimmed[:end]
 	}
 
+	firstWord = strings.TrimFunc(firstWord, func(r rune) bool { return !unicode.IsLetter(r) })
+
```

Comment:
I think this function makes the previous `IndexFunc` call unnecessary. Let's remove it.

Reply:
done